prog/victoria_metrics/victoria_metrics_setup.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class Prog::VictoriaMetrics::VictoriaMetricsSetup < Prog::Base
```

Comment:
This one is personal taste, so feel free to ignore, but I see setup as part of the original prog and usually embed there. One benefit is that I can follow the entire creation without jumping between files.

That said, other people prefer to split logic into multiple files, so it is indeed a matter of taste.

Reply:
Ack, I moved these into the same file to make things a bit simpler